common/vpc_stack.py

```diff
@@ -2,11 +2,13 @@
 
 from aws_cdk import (Stack,
     aws_ec2 as ec2,
+    aws_s3 as s3,
     Tags)
 
 from constructs import Construct
 
 VPC_CIDR_CONTEXT= "VPC_CIDR"
+FLOW_LOGS_BUCKET="sagebase-vpc-flow-logs-bucket-bucket-5lvxjv2gp37h"
```

Comment:
there are already a bunch of VARs setup in `cdk.json`, like `VPC_CIDR_CONTEXT` why not follow the pattern and put this there as well?